lib/cog/command/command_resolver.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Cog.Command.CommandResolver do
+
+  alias Cog.Repo
+  alias Cog.Queries.Command
+  alias Cog.Queries.Alias
```

Comment:
This gets kinda confusing to me if you're calling `Alias` but it's really a Query module. I always think it's the model module when I see `Alias`.